tests/kernel/install_kotd.pm

```diff
@@ -102,15 +103,27 @@ END
 
 sub run {
     my $self = shift;
+
+    my $baseurl;
+    my $default_url;
+    my $url;
+    my $version;
```

Comment:
Too many variables just for one fixed repository for openSUSE(which don't need any variable) and SLE which needs only `$version`.

Reply:
right. It could be done similar to this:

    if (is_sle) {
        $default_url = "http://download.suse.de/ibs/Devel:/Kernel:/SLE" . get_required_var("VERSION") . "/standard/";
    } else {
        $default_url = "http://download.opensuse.org/repositories/Kernel:/HEAD/standard/";
    }
    $url = get_var('KOTD_REPO', $default_url) .  '/' . get_required_var("ARCH") . '/';

Reply:
and remove declarations of variables (but not $default_url)

Reply:
@czerw I don't mind more variables if it helps readability, but going to change it a bit.